From ba7b1aba489c8c1903d992454d7ba1cee13b2005 Mon Sep 17 00:00:00 2001
From: "Thang Q. Nguyen" <thang@os.amperecomputing.com>
Date: Fri, 16 Oct 2020 04:09:46 +0000
Subject: [PATCH 5/9] bindings: ipmi: Add binding for Aspeed SSIF BMC driver

Add device tree binding document for the Aspeed SSIF BMC driver.

Signed-off-by: Thang Q. Nguyen <thangqn@amperecomputing.com>
---
 .../bindings/ipmi/aspeed-ssif-bmc.txt          | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)
 create mode 100644 Documentation/devicetree/bindings/ipmi/aspeed-ssif-bmc.txt

diff --git a/Documentation/devicetree/bindings/ipmi/aspeed-ssif-bmc.txt b/Documentation/devicetree/bindings/ipmi/aspeed-ssif-bmc.txt
new file mode 100644
index 000000000000..6d2aadd442da
--- /dev/null
+++ b/Documentation/devicetree/bindings/ipmi/aspeed-ssif-bmc.txt
@@ -0,0 +1,18 @@
+* Aspeed SSIF IPMI interface
+
+The Aspeed SOC (AST2500) is commonly used as BMCs
+(BaseBoard Management Controllers) and the IPMI SSIF interface can be used
+to perform in-band IPMI communication with their host.
+
+Required properties:
+
+- compatible : should be
+	"aspeed,ast2500-ssif-bmc"
+- reg: I2C address the registers
+
+Example:
+
+	ssif-bmc@10 {
+		compatible = "aspeed,ast2500-ssif-bmc";
+		reg = <0x10>;
+	};
-- 
2.18.4

